INFO     root:test01.py:26 正在执行调用执行数据：{'path': '/api/createorder', 'method': 'post', 'headers': {'token': '23657DGYUSGD126731638712GE18271H', 'content-type': 'application/json'}, 'param_type': 'json', 'params': {'cartid': 45233, 'openid': 'UEHUXUXU78272SDSassDD', 'productid': 8888, 'userid': 17890}, 'expect': {'data': [{'openid': 'UEHUXUXU78272SDSassDD', 'orderid': 23441289, 'orderprice': 129.9, 'productid': 8888, 'productnum': 1, 'userid': 17890}], 'httpstatus': 201, 'result': 'success'}, 'x_y': [6, 11]}
INFO     root:api.py:45 正在调用运行接口方法
INFO     root:api.py:26 正在调用 post请求方法
INFO     root:common_assert.py:7 正在调用断言公共方法
ERROR    root:test01.py:39 错误，原因：错误！响应 code：200 预期 code：201